Modify rule S5135: Clarify safe loader (#3089)
SonarSource/rspec: rules/S5135/python/how-to-fix-it/yaml.adoc

prevents the execution of arbitrary or dangerous functions.

Note that the same level of security can be reached with the `load` method as
soon as the `Loader` component `SafeLoader` is passed to it. All other loaders

> are considered insecure, as they may allow to deserialize objects.

Reviewer: Just a small phrasing suggestion:
```suggestion
are considered insecure, as they may allow deserialization of objects.
```
